lib/components/TrayPortal.js

```diff
@@ -129,6 +130,9 @@ export default React.createClass({
   open() {
     focusManager.markForFocusLater();
     this.setState({isOpen: true}, () => {
+      if (this.props.onOpen) {
```

Comment:
With this in place (assuming we want to use this to set focus to some other element), focus will be set to the content and then to the other element, so I don't know if this is great for setting focus.